app/controllers/student_teams_controller.rb

```diff
@@ -51,8 +51,7 @@ def view
         break
       end
     end
-
-    @teammate_review_allowed = true if @student.assignment.find_current_stage=='Finished' || @current_due_date&&(@current_due_date.teammate_review_allowed_id ==3 ||@current_due_date.teammate_review_allowed_id ==2) #late(2) or yes(3)
+    @teammate_review_allowed = true if @current_due_date&&(@current_due_date.teammate_review_allowed_id ==3 ||@current_due_date.teammate_review_allowed_id ==2) #late(2) or yes(3)
```

Comment:
I am not sure who is the owner of this change. In OSS project, we changed only Invitations and JoinTeamRequest controllers. These changes were done by some other team and their changes must have been merged before us.